packages/pwa-buildpack/lib/TestHelpers/testTargets/MockedBuildBus.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * A mock BuildBus for testing target integrations. Instead of using the local
+ * `package.json` to detect and order the pertaining dependencies, this takes
+ * a set of pre-ordered dependencies that can include "virtual dependency"
+ * objects.
+ *
+ * @module Buildpack/TestHelpers
+ *
+ */
+const BuildBus = require('../../BuildBus/BuildBus');
+const { ExplicitDependency, resolver } = require('pertain');
+class MockedBuildBus extends BuildBus {
+    /** @public */
+    static clear() {
+        throw new Error(
+            'MockedBuildBus.clear() not supported. More details at https://twitter.com/JamesZetlen/status/1244683087839137792'
```

Comment:
Would we be better of having these documented in dev docs and pointing to them? I love tweets as documentation but secretly.... I don’t haha

Reply:
Yeah, we should be adapting the JSDoc comments, which is something I'm talking about with @jcalcaben . The rest of the stuff is easter eggs

Reply:
Yeah we can create a best practices page for testing that includes these kinds of notes

Reply:
Yeah, the only tweet I've ever linked in a code comment has since been deleted.